qipr_approver/approver/models/models.py

```diff
@@ -89,7 +90,8 @@ class Person(Provenance, Registerable):
     user = models.OneToOneField(User, null=True, on_delete=models.CASCADE, related_name="person")
     webpage_url = models.CharField(max_length=50, null=True)
     title = models.CharField(max_length=50, null=True)
-    department = models.CharField(max_length=50, null=True)
+    department = models.ForeignKey(ClinicalDepartment, null=True, on_delete=models.SET_NULL,
+                                   related_name="person")
```

Comment:
We might need to check what departments are in the db currently and maybe store them somehow so we don't lose user data